File: lecture01/exercise01-A/solution.cpp
// Exercise 01-A: Line Totals

#include <iostream>
#include <string>
#include <sstream>

using namespace std;

int main(int argc, char *argv[]) {
    string line;

    while (getline(cin, line)) {
    	stringstream ss(line);

    	int total = 0;
    	int n;
    	while (ss >> n) {
    	    total += n;
	}

	cout << total << endl;
    }

    return 0;
}

// vim: set sts=4 sw=4 ts=8 ft=cpp:

File: lecture01/exercise01-A/solution.py
#!/usr/bin/env python3

'''
Exercise 01-A: Line Totals
'''

import sys

def main():
    for line in sys.stdin:
        total = sum(map(int, line.split()))
        print(total)

if __name__ == '__main__':
    main()

# vim: set sts=4 sw=4 ts=8 expandtab ft=python:
